config/compose.toml

```diff
@@ -21,9 +21,10 @@ log_level = "info"
 
 enable_schema_caching = true
 
+# Bolt Configuration
 [services]
-storage = "bolt"
 service_endpoint = "http://localhost:8080"
```

Comment:
let's think how this plays in with docker compose. i believe we should consider reading in config through env vars as an override for config set here, where docker compose would set this as an env var

Reply:
created issue for this - https://github.com/TBD54566975/ssi-service/issues/233